js-foundations.js

```diff
@@ -3,27 +3,27 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
-const isEvenlyDivisbleByThreeOrNot = num => (num % 3 === 0) ? `${num} is divisible by three` : `${num} is not divisble by three`
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
-const helloMeGenerator = (firstName, lastName, age, hobby) => {return {firstName, lastName, age, hobby} }
-const ben = helloMeGenerator('benjamin','geyer',30, 'juggling')
+
+
 
 // ------------------------------------- Consider the variable:
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 
 // 3a. Write a function that returns every other item from the array as one string. Expected output: "streetlamp teeth nephew database"
 
-const everyOtherItemFromTheArrayAsOneStringGenerator = arr => arr.filter((elm, ind) => (ind+2) % 2 === 0).join(' ')
+
 
 // 3b. Create a function that takes in the variable and returns the array with all the words capitalized. Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
-const arrayWithAllTheWordsCapitalizedGenerator = arr => arr.map(word => word[0].toUpperCase()+word.slice(1))
+
 
 // 3c. STRETCH: Create a function that returns the length of each string from the randomNouns variable. Expected output: [10, 6, 5, 10, 6, 11, 8]
-const lengthOfStringInArray = arr => arr.map(word => word.length)
+
+
 
```

Comment:
Great one line functions! Next time will you include some console logs for me, please?